index.html

```diff
@@ -330,40 +339,25 @@ <h3 class="languages">Languages</h3>
             <div class="event17">
               <h3>Frameworks</h3>
               <button type="submit" class="dropdown">
-                <img src="./images/Icon-Chevron-Right-Larger.png" alt="" />
+                <img src="./images/Disabled.png" alt="" />
```

Comment:
Kindly check this image path 😉, the image is not displaying.
![img](https://user-images.githubusercontent.com/73854999/199366472-1f06f1e0-9aef-4ee0-b47a-145e5b3794d2.jpg)